### DISCUSSION TITLE
Is possible pause/destroy a instance without stop the video DOM with the webcam videostream?

### DISCUSSION THREAD

**Discussion by wlauton** [jeeliz/jeelizFaceFilter#295] [Q&A]
I was trying make some multiple effects gui, calling a function to render a 3d cube like boilerplate and another one rendering a sphear for example. I create a button to call:
JEELIZFACEFILTER.toggle_pause(true,false); but the theeCanvas don't get clear, just pause the rendering, and when i call JEELIZFACEFILTER.destroy(); the video of the webcam of the videoElement DOM gets black, what could i do to call diferent functions to render diferents models on face and clean the render canvas by demand?

**Comment by xavierjs** [comment]
Hi,

What do you want to do? Your message is really not clear and badly written, both in the form and the content.

You should destroy JeelizFaceFilter if you unmount the component / quit the app
If you just hide the component, you can just pause it.

Jeeliz FaceFilter does not manage your three.js scene. We provide some helpers to manage the three.js scene but if you want to clear it, you need to use the right way to do it with Three.js.
If you want some useful help, please ask more specific questions. Also learn how three.js works before using this library. It looks that your issue is more related to Three.js than to Jeeliz FaceFilter.

If you want to toggle different filters matching different Three.js scenes, the best way to do it is to clear all objects of the scene, fill the scene, then if the user picks another filter you clean the scene and repopulate it. You don't need to stop or destroy Jeeliz Facefilter to do it.
